src/Component/JsonSchemaCompiler/Tests/CompilerTest.php

```diff
@@ -58,7 +58,7 @@ public function testNamingDuplicates(): void
         self::assertNotNull($model);
         self::assertInstanceOf(Property::class, $sandboxProperty = $model->getProperty('sandbox'));
         self::assertInstanceOf(ObjectType::class, $sandboxProperty->type);
-        self::assertEquals('OpenBankingTrackerSandbox', $sandboxProperty->type->className);
+        self::assertEquals('OpenBankingTrackerSandbox1', $sandboxProperty->type->className);
```

Comment:
Why does it have an extra "1"? If it's because we have an `OpenBankingTrackerSandboxEnum` enum, for me it's KO, we should accept to have an `OpenBankingTrackerSandbox` model and an `OpenBankingTrackerSandboxEnum` enum, possible solution: split the check conflicts table for templates and enums?

Reply:
This test was wrong and it has nothing to do with enums. The class name generated by the ModelResolver was not correctly used when defining the property type in the ObjectGuesser (see https://github.com/Korbeil/jane-v8/pull/1/files#diff-1d6b3462aea475ebdb9f60465f4804ced91fd54a0f616898b3f5de1250ff1778R25). Now both the issue and the test are fixed.